lib/constants/index.ts

```diff
@@ -254,3 +254,5 @@ export const verra = {
   projectSearch: `${VERRA_REGISTRY_API}/resource/resource/search?maxResults=2000&$count=true&$skip=0&$top=50`,
   projectDetailPage: `${VERRA_REGISTRY}/app/projectDetail/VCS`, // add ID after VCS like /191
 };
+export const IS_PRODUCTION =
+  process.env.NEXT_PUBLIC_VERCEL_ENV === "production";
```

Comment:
I would prefer that `lib` stay completely agnostic about nextjs or vercel, so we will have to pass this flag down at runtime

Reply:
Okay